ocaml/license/dune

```diff
@@ -1,23 +1,21 @@
-(* -*- tuareg -*- *)
-let coverage_rewriter =
-  match Sys.getenv "BISECT_ENABLE" with
-  | "YES" -> "(preprocess (pps bisect_ppx -conditional))"
-  | _ -> ""
-| exception Not_found -> ""
+(library
+  (name daily_license_check)
+  (modules daily_license_check)
+  (libraries
+    xapi-client
+    xapi-stdext-date
+  )
+)
 
-let () = Printf.ksprintf Jbuild_plugin.V1.send {|
 (executable
   (name daily_license_check_main)
   (public_name daily-license-check)
   (package xapi)
-  (modules (:standard \ test_daily_license_check))
+  (modules daily_license_check_main)
```

Comment:
If you moved the file that was excluded then you can just remove this line completely.

Reply:
Since I'm building a library with the other ml file in the folder I can't remove this line.